Makefile

```diff
@@ -131,8 +128,7 @@ test-source-headers:
 	@scripts/validate-license.sh
 
 .PHONY: test-acceptance
-test-acceptance: TARGETS = linux/amd64
-test-acceptance: build build-cross
+test-acceptance: build build-linux-amd64
```

Comment:
`test-acceptance` previously built current arch + all architectures. Now it is building current + linux/amd64. 

If I'm not mistaken, we previously did this to:
- build all architectures to ensure all architectures build
- build current arch, to run acceptable tests

We need to continue the above behavior here (rather than building current + linux/amd64 only; runners are most likely to be linux/amd64, which makes this redundant too)